primeqa/ir/dense/colbert_top/colbert/infra/config/base_config.py

```diff
@@ -41,48 +37,45 @@ def from_deprecated_args(cls, args):
     @classmethod
     def from_path(cls, name):
 
-        print_message(f"#> base_config.py from_path {name}")
+        print_message(f"#> base_config.py from_path: {name}")
+        with open(name, "r", encoding="utf-8") as filename:
+            args = ujson.load(filename)
 
-        with open(name) as f:
+            if "config" in args:
+                args = args["config"]
 
-            args = ujson.load(f)
+        return cls.from_deprecated_args(
+            args
+        )  # the new, non-deprecated version functions the same at this level.
 
-            print_message(f"#> base_config.py from_path args loaded! ")
-
-            if 'config' in args:
-                args = args['config']
-
-                print_message(f"#> base_config.py from_path args replaced ! ")
-
-
-        return cls.from_deprecated_args(args)  # the new, non-deprecated version functions the same at this level.
-    
     @classmethod
     def load_from_checkpoint(cls, checkpoint_path):
-        if checkpoint_path.endswith('.dnn') or checkpoint_path.endswith('.model'):
```

Comment:
I'm not sure about switching to double quotes everywhere, even when the string literal is a filename or a part of it.  It will create conflicts with the other pending PRs and other work in progress, and reduce compatibility with the Stanford code.